src/main/java/Event.java

```diff
@@ -0,0 +1,14 @@
+public class Event extends Task{
+
+    private String time;
+
+    public Event(String name, String time) {
+        super(name);
+        this.time= time;
```

Comment:
Same as in Duke.java line 94, you may want to add a space between “this.time” and “=“ to make it more consistent.